cli/test/README.md

```diff
@@ -33,7 +33,7 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set log`|**PENDING** (currently `Log module configuration not yet supported`)|N/A|
 |`fastddsqosprof file.xml set participant`|`ERROR: participant requires a profile name` [^1]|`PARTICIPANT_USAGE`|
 |`fastddsqosprof file.xml set topic`|**PENDING** (currently `Topic configuration not yet supported`)|N/A|
-|`fastddsqosprof file.xml set transport_descriptor`|**PENDING** (currently `Transport descriptor configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor`|`ERROR: transport_descriptor requires a profile name` [^1]|N/A|
```

Comment:
Shall I remove the `[^1]`?

Reply:
Yep. That specific footnote only applies for the `default_profile` attribute. There is no default profile attribute applying to Transport Descriptor, so the footnote here is not correct.